### PR TITLE
solana: fix upgrade manager

### LINKED ISSUE #25: solana: finish upgrade manager program
It was left in shambles. Wrap this up.

### PR DESCRIPTION
**Pull request by a5-pickle**
Deployed upgrade manager to Solana devnet. Forked tests work.

Closes #25.

### CODE REVIEW

File: solana/programs/upgrade-manager/src/composite/mod.rs [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+        seeds = [program.key().as_ref()],
+        bump = receipt.program_data_bump,
+        seeds::program = bpf_loader_upgradeable::id(),
+    )]
+    program_data: Account<'info, ProgramData>,
+}
+
+#[derive(Accounts)]
+pub struct RequiredSysvars<'info> {
+    /// Wormhole Core Bridge needs the clock sysvar based on its legacy implementation.
```
Comment (gator-boi) [inline solana/programs/upgrade-manager/src/composite/mod.rs:129]: Stale comment?

File: solana/programs/upgrade-manager/src/composite/mod.rs [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    ///
+    /// CHECK: Must equal clock ID.
+    #[account(address = solana_program::sysvar::clock::id())]
+    pub clock: AccountInfo<'info>,
+
+    /// Wormhole Core Bridge needs the rent sysvar based on its legacy implementation.
```
Comment (gator-boi) [inline solana/programs/upgrade-manager/src/composite/mod.rs:135]: Here too?

Comment (a5-pickle) [reply solana/programs/upgrade-manager/src/composite/mod.rs]: Fixed here: https://github.com/wormhole-foundation/example-liquidity-layer/pull/38/commits/ba7ab69cb4294b58d714ba501e63f2382e9e3f74

File: solana/programs/upgrade-manager/src/composite/mod.rs (hunk 1 above)
Comment (a5-pickle) [reply solana/programs/upgrade-manager/src/composite/mod.rs]: Fixed here: https://github.com/wormhole-foundation/example-liquidity-layer/pull/38/commits/ba7ab69cb4294b58d714ba501e63f2382e9e3f74

**Review by gator-boi — APPROVED** [review]

**Pull request merged**